test/api/liskApi.js

```diff
@@ -695,4 +691,31 @@ describe('Lisk.api()', () => {
 			});
 		});
 	});
+
+
+	describe('#constructRequestData', () => {
+		it('should construct optional request data for API helper functions', () => {
+			const address = '123';
+			const requestData = {
+				limit: '123',
+				offset: 5,
+			};
+			const expectedObject = {
+				address: '123',
+				limit: '123',
+				offset: 5,
+			};
+			const createObject = privateApi.constructRequestData({ address }, requestData);
+			(createObject).should.be.eql(expectedObject);
+		});
+
+		it('should construct with variable and callback', () => {
+			const address = '123';
+			const expectedObject = {
+				address: '123',
+			};
+			const createObject = privateApi.constructRequestData({ address }, () => { return '123'; });
```

Comment:
Prefer `() => '123'` arrow body shorthand when there's just a single return statement. But using '123' here is confusing since we already have it for `address`.